jcm/diffusion.py

```diff
@@ -4,19 +4,41 @@
 
 @tree_math.struct
 class DiffusionFilter:
-    tendency_diff_timescale: float # Diffusion timescale (s)
-    tendency_diff_order: int # Order of diffusion operator for tendencies
-    state_diff_timescale: float # Diffusion timescale (s)
-    state_diff_order: int  # Order of diffusion operator for state variables
+    vor_q_timescale: jnp.float32 # Diffusion timescale (s)
```

Comment:
Is there a reason we have to specify the exact type? I think there are `jnp.float_ ` types we can use to let the JAX configuration decide? Just in case it causes problems with different devices